accounts/keystore/account_cache.go

```diff
@@ -212,7 +212,9 @@ func (ac *accountCache) maybeReload() {
 	ac.watcher.start()
 	ac.throttle.Reset(minReloadInterval)
 	ac.mu.Unlock()
-	ac.scanAccounts()
+	if err := ac.scanAccounts(); err != nil {
+		// ignore error
```

Comment:
`scanAccounts` logs all errors internally. It'd be better to remove the error return value from the method.